packages/devextreme/js/ui/menu/ui.menu.js

```diff
@@ -271,6 +271,23 @@ class Menu extends MenuBase {
         this._initAdaptivity();
     }
 
+    _isTargetOutOfComponent(relatedTarget) {
+        const isInsideRootMenu = $(relatedTarget).parents(`.${DX_MENU_CLASS}`).length !== 0;
```

Comment:
this functionality is not tested

Reply:
[d90a6b7f24312a9901977c1207f33c86766ffe5c]